src/App.css

```
@@ -0,0 +1,172 @@
+@import url("./api/normalize.css");
+
+body {
+  font-family: Helvetica, neue;
+  background-color: rgb(245, 245, 245);
+}
+
+.toDos {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  justify-items: center;
+}
+
+.toDos:focus {
+  outline: none;
+}
+
+.toDos-header {
+  font-size: 100px;
+  font-weight: normal;
+  color: rgba(175, 47, 47, 0.15);
+  margin: 0;
+}
+
+#root {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.input-base {
+  width: 600px;
+  padding: 16px 16px 16px 60px;
+  border: none;
+  font-size: 24px;
+  line-height: 1.4em;
+  box-shadow: inset 0 -2px 1px rgba(0, 0, 0, 0.03);
+}
+
+.input-base::placeholder {
+  opacity: 0.2;
+  font-style: italic;
+}
+
+.toDos--dropDownTask {
```

If you follow BEM methodology, you should know that `--` is used for modificators, but not for the elements.
```css
.block {}
.block_modificator {}
// or
.block--modificator{}

.block__element {}
.block__element_modificator {}
// or
.block__element--modificator {}
```
```suggestion
.todos__dropdownTask {
```